include/vk_mem_alloc.h

```diff
@@ -6069,6 +6091,91 @@ class VmaMappingHysteresis
 
 #endif // _VMA_MAPPING_HYSTERESIS
 
+#if VMA_EXTERNAL_MEMORY_WIN32
+class VmaWin32Handle
+{
+public:
+    VmaWin32Handle() noexcept : m_hHandle(VMA_NULL) { }
+    VmaWin32Handle(HANDLE hHandle) noexcept : m_hHandle(hHandle) { }
```

Comment:
It is a good practice to add `explicit` before every constructor that takes 1 argument to avoid implicit conversions.

Reply:
tru